draft-lmbdhk-quic-multipath.md

```diff
@@ -256,6 +256,71 @@ Senders MUST manage per-path congestion status, and MUST NOT send more data on a
 
 Multipath TCP uses the LIA congestion control scheme specified in {{RFC6356}}.  This scheme can immediately be adapted to Multipath QUIC. Other coupled congestion control schemes have been proposed for Multipath TCP such as {{OLIA}}.
 
+# Path States
+
+{{fig-path-states}} shows the states that an endpoint's path can have.
+
+~~~
+       o
+       | Initiate path validation
+       | PATH_CHALLENGE received on new path
+       v
+ +------------+    Path validation abandoned
+ | Validating |----------------------------------+
+ +------------+                                  |
+       |                                         |
+       | Path validation succeeded               |
+       |                                         |
+       v        Associated CID have been retired |
+ +------------+        Path's idle timeout       |
+ |   Active   |----------------------------------+
+ +------------+                                  |
+       |                                         |
+       | PATH_ABANDONED sent/received            |
+       v                                         |
+ +------------+                                  |
+ |   Closing  |                                  |
+ +------------+                                  |
+       |                                         |
+       | Associated CID have been retired        |
+       | Path's idle timeout                     |
+       v                                         |
+ +------------+                                  |
+ |   Closed   |<---------------------------------+
+ +------------+
+~~~
+{: #fig-path-states title="States of a path"}
+
+In non-terminal states, hosts MUST track the following information.
+
+- Associated 4-tuple: The tuple (source IP, source port, destination IP, destination port)
+used by the endhost to send packets over the path.
+
+- Associated Destination Connection ID: The Connection ID used to send packets over the path.
+
+If multiple packet number spaces are used over the connection, hosts MUST also track the following information.
+
+- Path Packet Number Space: The endpoint maintains a separate packet number for sending and receiving packets over this path.
+Packet number considerations described in {{QUIC-TRANSPORT}} apply within the given path.
+
+In the "Active" state, hosts MUST also track the following information.
+
+- Associated Source Connection ID: The Connection ID used to receive packets over the path.
+
+A path in the "Validating" state performs path validation as described in {{Section 8.2 of QUIC-TRANSPORT}}.
+An endhost SHOULD NOT send non-probing frames on a path in "Validating" state, as it has no guarantee that packets
+will actually reach the peer.
+
+The endhost can use all the paths in the "Active" state, provided that their usage is enabled by the path's congestion
```

Comment:
"usage is enabled by" is a bit unclear. Maybe: ... provided the congestion control and flow control currently allow sending of new data on a path